plugins/catalog-backend/src/ingestion/processors/GithubOrgReaderProcessor.ts

```diff
@@ -71,14 +60,14 @@ export class GithubOrgReaderProcessor implements CatalogProcessor {
       return false;
     }
 
-    const client = await this.createClient(location.target);
+    const { client, tokenType } = await this.createClient(location.target);
```

Comment:
It might be kind of nice if you could control which token type is used from the credentials provider. For instance, if your `integrations.github` has both a `token` and `app` configured then you could say use the `token` for this processor because we know it will work with emails. However, I understand this would probably complicate the credentials provider code and require an ordered lookup since one or the other may also exist.

Reply:
Yeah I guess the other thing is that we really don't want to recommend the usage of tokens either. And the fact that hopefully with the new catalog work, we can get around this issue by querying users individually to get their email from GitHub rather than grabbing it from the `ReaderProcessor` 👍

Reply:
Makes sense. For us, the email field (as is) will provide little value so it's not a big deal either way.

Reply:
Yeah it's a good point Andrew but I also think it will add too much to the CredentialsProvider. The long term fix is perhaps to update the org reader to catch the error and set the email as null or retry with a personal token. It feels like GitHub does not want users rely on emails from their API.